pages/index.js

```diff
@@ -1,24 +1,129 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import Head from "next/head";
+import Image from "next/image";
+import styles from "../styles/Home.module.css";
+import { useState } from "react";
+
+const DECODE_PREFIX = "data:image/png;base64,";
+const API = "http://127.0.0.1:8000/photo/";
+
+async function getSubmissionListWithContest() {
+  return fetch(API + "submissions/", {
+    method: "GET",
+    headers: {
+      "Content-Type": "application/json",
+    },
+  }).then((data) => data.json());
+}
+
+async function getUserList() {
```

Comment:
maybe have a single file to make requests only, export them, and import here to use

Reply:
this was in André's PR, maybe he already did it?

Reply:
Remember to pull when @andrevieira11 fixes this

Reply:
done